registry-photon/v2.10.2/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Unless they require modifications, I'd strongly suggest [we copy these file straight from the repo](https://github.com/canonical/harbor-rocks/pull/3/files#diff-5b302c1a9a8131a676a069e06e17ae2d4f5f2d1f54b79165a7069b401edc1c34R124-R125) instead of duplicating them.

Reply:
Those scripts are really straight forward, I'd rather keep it simple. Tbh, we could even drop install-cert.sh since it's a no-op on non-photon images.

Reply:
Anyway, I can do a cleanup afterwards. Currently focusing on the integration tests.